packages/react-form/README.md

```diff
@@ -427,7 +427,7 @@ return <TextField label="Title" {...title} />;
 
 #### `useChoiceField()`
 
-An extension to `useField()` that produces a new field compatible with `<Checkbox />` and `<RadioButton />` from `@shopify/polaris`.
+An extension to `useField()` that produces a new field compatible with `<Checkbox />` and `<RadioButton />` from `@shopify/polaris`. Note that this hook cannot be used within a `useForm`, `useChoiceField` can only be used for standalone fields.
```

Comment:
upon topatting it was discovered that `useChoiceField` is no longer compatible with `useForm` compositions, so I am adding this extra note to clarify that case. i.e.,

```
useForm({
  fields: {
    // this field will not be compatible with the FieldBag type for fields above
    flag: useChoiceField(true),
  },
});
```